tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -23,6 +26,49 @@ TEST(RNTuple, CreateField)
    field->DestroyValue(value);
 }
 
+TEST(RNTuple, StdPair)
+{
+   auto field = RField<std::pair<int64_t, float>>("pairField");
+   EXPECT_STREQ("std::pair<std::int64_t,float>", field.GetType().c_str());
+   auto otherField = RFieldBase::Create("test", "std::pair<int64_t, float>").Unwrap();
+   EXPECT_STREQ(field.GetType().c_str(), otherField->GetType().c_str());
+   EXPECT_EQ((sizeof(std::pair<int64_t, float>)), field.GetValueSize());
+   EXPECT_EQ((sizeof(std::pair<int64_t, float>)), otherField->GetValueSize());
+   EXPECT_EQ((alignof(std::pair<int64_t, float>)), field.GetAlignment());
+   EXPECT_EQ((alignof(std::pair<int64_t, float>)), otherField->GetAlignment());
+
+   auto pairPairField = RField<std::pair<std::pair<int64_t, float>,
+      std::vector<std::pair<CustomStruct, double>>>>("pairPairField");
+   EXPECT_STREQ(
+      "std::pair<std::pair<std::int64_t,float>,std::vector<std::pair<CustomStruct,double>>>",
+      pairPairField.GetType().c_str());
+
+   FileRaii fileGuard("test_ntuple_rfield_stdpair.root");
+   {
+      auto model = RNTupleModel::Create();
+      auto pair_field = model->MakeField<std::pair<double, std::string>>(
+         {"myPair", "a very cool field"}
+      );
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), "pair_ntuple", fileGuard.GetPath());
+      for (int i = 0; i < 100; i++) {
```

Comment:
I'm not sure that writing 100 objects in 10 clusters adds much to the test. Consider writing only 1 or 2.